[2024-10-08T13:19:31.839+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T13:19:31.857+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_etl_05.run_extract_container_2 manual__2024-10-08T13:19:18.047208+00:00 [queued]>
[2024-10-08T13:19:31.865+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_etl_05.run_extract_container_2 manual__2024-10-08T13:19:18.047208+00:00 [queued]>
[2024-10-08T13:19:31.865+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-08T13:19:31.877+0000] {taskinstance.py:2888} INFO - Executing <Task(DockerOperator): run_extract_container_2> on 2024-10-08 13:19:18.047208+00:00
[2024-10-08T13:19:31.885+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=170) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T13:19:31.886+0000] {standard_task_runner.py:72} INFO - Started process 171 to run task
[2024-10-08T13:19:31.887+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_etl_05', 'run_extract_container_2', 'manual__2024-10-08T13:19:18.047208+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpasusvu0_']
[2024-10-08T13:19:31.888+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask run_extract_container_2
[2024-10-08T13:19:31.931+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_etl_05.run_extract_container_2 manual__2024-10-08T13:19:18.047208+00:00 [running]> on host ade6ae45e28b
[2024-10-08T13:19:32.007+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='BB' AIRFLOW_CTX_DAG_ID='dag_etl_05' AIRFLOW_CTX_TASK_ID='run_extract_container_2' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T13:19:18.047208+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T13:19:18.047208+00:00'
[2024-10-08T13:19:32.008+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T13:19:32.030+0000] {docker.py:367} INFO - Starting docker container from image etl
[2024-10-08T13:19:32.032+0000] {docker.py:375} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-10-08T13:19:32.596+0000] {docker.py:438} INFO - Category          Name         Author  ... Rate_Number  Date_First  Like_First
[2024-10-08T13:19:32.597+0000] {docker.py:438} INFO - 0  Romance  Lore Olympus  Rachel Smythe  ...        9.58  2018-03-04      405348
[2024-10-08T13:19:32.597+0000] {docker.py:438} INFO - 
[2024-10-08T13:19:32.598+0000] {docker.py:438} INFO - [1 rows x 9 columns]
[2024-10-08T13:19:32.959+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T13:19:32.960+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_etl_05, task_id=run_extract_container_2, run_id=manual__2024-10-08T13:19:18.047208+00:00, execution_date=20241008T131918, start_date=20241008T131931, end_date=20241008T131932
[2024-10-08T13:19:32.982+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T13:19:33.002+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-08T13:19:33.003+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
